docs/contribute/plugin/intro.md

```diff
@@ -77,14 +77,14 @@ There are several plug-in releases with the WasmEdge official releases. Please c
 
 | Plug-in | Rust Crate | Released Platforms | Build Steps |
 | --- | --- | --- | --- |
-| WasmEdge-Process | [wasmedge_process_interface][] | `manylinux2014 x86_64`, `manylinux2014 aarch64`, and `ubuntu 20.04 x86_64` (since `0.10.0`) | [Build Wtih WasmEdge-Process](../source/plugin/process.md) |
+| [WasmEdge-Process](https://wasmedge.org/docs/contribute/source/plugin/process) | [wasmedge_process_interface][] | `manylinux2014 x86_64`, `manylinux2014 aarch64`, and `ubuntu 20.04 x86_64` (since `0.10.0`) | [Build Wtih WasmEdge-Process](../source/plugin/process.md) |
```

Comment:
Looks like this needs to be corrected. 
Take WASI-NN with OpenVINO backend as an example. The link should be set to a document page of the corresponding plugins.

```
[WASI-NN with OpenVINO backend](../../develop/rust/wasinn/openvino.md)
```

This should raise an issue that indicates that there are some plugins without document pages. And we are going to write them in the other PRs.

Reply:
So Should I change the name of the file from `process.md` to `wasmedge-process.md`

Reply:
No need to do this.

Reply:
So should I remove the link for now or can we use this link https://wasmedge.org/docs/contribute/source/plugin/process/#build-wasmedge-with-wasmedge-process-plug-in-1?

Reply:
You should never use the generated URLs. The original related path is the best practice.

Reply:
Sure, I'll keep that in mind. Then for now should I remove link since we do not have a document regarding wasmedge-process?

Reply:
You can keep the link, instead of https:// use the ../ an example is in the 80th line itself. In a gist, we tend to use relative path for the docs here

Reply:
No probs, no questions are lame or silly :)

Reply:
Should I also use relative paths for the links I have added under the `docs/start/wasmedge/extensions/plugins.md`?